cafebabel/static/js/textarea-change.js

```diff
@@ -1,6 +1,7 @@
 /* Auto-expand textarea when typing http://jsfiddle.net/hmelenok/WM6Gq/ */
 Array.from(document.querySelectorAll('textarea')).forEach(textArea => {
   function resize() {
+    textArea.style.height = 'auto'
```

Comment:
Because it is unusual to see, I'd rather read a comment in the code that tells _why_ we need to hack it so we're not tempted to clean it off in the next refactoring ;)

Reply:
ok !